packages/fluentui/react-northstar/src/components/Pill/PillImage.tsx

```diff
@@ -0,0 +1,92 @@
+import { Accessibility } from '@fluentui/accessibility';
+import {
+  ComponentWithAs,
+  getElementType,
+  useUnhandledProps,
+  useAccessibility,
+  useFluentContext,
+  useStyles,
+  useTelemetry,
+} from '@fluentui/react-bindings';
+import * as React from 'react';
+
+import { UIComponentProps, commonPropTypes, SizeValue } from '../../utils';
+import { FluentComponentStaticProps } from '../../types';
+
+export interface PillImageProps extends UIComponentProps {
+  /** Accessibility behavior if overridden by the user. */
+  accessibility?: Accessibility<never>;
+
+  /**
+   * A PillImage can be sized.
+   */
+  size?: Extract<SizeValue, 'smaller' | 'small' | 'medium'>;
+
+  /** PillImage source URL. */
+  src?: string;
```

Comment:
Why is this prop optional ? What is the expected behaviour if `src` is not passed to this component ?

Reply:
Same behavior that `Image` has